server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -54,8 +55,8 @@ channels{TECHNICAL}
     {
       String input = _input
                      .getText(Interval.of(_tokenStartCharIndex, _input.index()))
-                     .replace("\r","").replace("\n","").trim();
-      if(StringUtils.isBlank(input.substring(5, input.length()-1)))
+                     .replace("\r","").replace("\n","").replace(".","").trim();
```

Comment:
Try using [`\R`](https://stackoverflow.com/questions/11644100/regular-expression-does-not-match-newline-obtained-from-formatter-object) here.

Reply:
Done.

Reply:
Wow! Didn't know about this feature.